devices/stm32f446.yaml

```diff
@@ -130,3 +130,4 @@ _include:
  - ../peripherals/fsmc/fsmc_sd.yaml
  - ../peripherals/tim/tim_ccm_v1.yaml
  - ../peripherals/sai/sai.yaml
+ - common_patches/otg_hs_remove_prefix.yaml
```

Comment:
I don't think it's important, but I placed patch lines between `- common_patches/rtc/rtc_bkpr.yaml` and `- common_patches/tim/tim_ccr.yaml` for consistency.

Reply:
Moved :)